test/kubemark/start-kubemark.sh

```diff
@@ -37,23 +39,39 @@ source "${KUBE_ROOT}/cluster/kubemark/util.sh"
 
 KUBECTL="${KUBE_ROOT}/cluster/kubectl.sh"
 KUBEMARK_DIRECTORY="${KUBE_ROOT}/test/kubemark"
-RESOURCE_DIRECTORY="${KUBEMARK_DIRECTORY}/resources"
+export RESOURCE_DIRECTORY="${KUBEMARK_DIRECTORY}/resources"
+export SHARED_CA_DIRECTORY=${SHARED_CA_DIRECTORY:-"/tmp/shared_ca"}
+export SCALEOUT_TP_COUNT="${SCALEOUT_TP_COUNT:-1}"
+export HAPROXY_TLS_MODE=${HAPROXY_TLS_MODE:-"bridging"}
```

Comment:
Can we instead use ENABLE_INSECURE_SCALEOUT_CLUSTER_MODE?  It's simpler for the user and not tied to HAProxy. We can set this to bridging or offloading based on ENABLE_INSECURE_SCALEOUT_CLUSTER_MODE = false/true.

Reply:
i have added a variable USE_INSECURE_SCALEOUT_CLUSTER_MODE to control how the components connect to the API servers. the bridging and the offloading mode is dedicated to the HAPROXY and the naming even inherited from the HAPROXY.

Reply:
Shouldn't HAPROXY_TLS_MODE be set to offloading when USE_INSECURE_SCALEOUT_CLUSTER_MODE=true?

Reply:
as explained above. those two controlls are with different purpose.

Reply:
Issue #1016 https://github.com/CentaurusInfra/arktos/issues/1016 to track the potential further enhancement.